deployment/docker-compose.dev.yml

```diff
@@ -115,6 +115,22 @@ services:
     networks:
       - xyne
 
+  livekit:
```

Comment:
@MayankBansal2004 can you aslo add this in deployment/portal/docker-compose.infra.yml too 
Also is there any mount that is needed in livekit to persist data?